### ISSUE TITLE
Advanced option to manually select which keys are shown

### ISSUE THREAD

**Issue by Aariq** [keycastr/keycastr#247]
I'd ideally love to show command keys, tab, and option + arrow keys (which doesn't show up currently with command keys only).  Would it be possible to just make an advanced option to select specific keys to be shown?

**Comment by akitchen** [comment]
Yes, this would be very useful. I think this is a theme across some other issues but thanks for opening an issue focused on it.

**Comment by Aariq** [comment]
Related issues: #289, #284, #279

**Comment by jmuheim** [comment]
Yes, PLEASE add this behaviour, it would be extremely useful! Thank you.

**Comment by koraysels** [comment]
this is going to be a fantastic addition, as this will enable my students to see the key combo's in realtime while I'm demoing something in an IDE..

**Comment by peterhgruber** [comment]
Especially `TAB` is important to show code completion or command completion with bash or any copilot.

**Comment by akitchen** [comment]
Would an option to add navigation keys to the current selected display options cover this need? An open-ended allow/deny list would be quite a bit more complicated though certainly not impossible. See #296 for current thoughts on this.

**Comment by Aariq** [comment]
I'm very pleased with the latest release that allows option + arrows to be displayed, however I wish I didn't also have to display every time I type a capital letter (shift + anything). So now the problem is more that I want to turn _off_ certain modifier keys (i.e. shift on its own).  And I guess this also doesn't cover TAB, mentioned above.

**Comment by raghubetina** [comment]
> So now the problem is more that I want to turn off certain modifier keys (i.e. shift on its own). And I guess this also doesn't cover TAB, mentioned above.

I very much agree with this. I love everything about the current Keycastr except that I can't find a way to turn off e.g. Shift+Letter every time I type a capital letter, or Shift+2 when trying to type an @ 

Can anyone advise if I'm overlooking a solution to this? Thanks very much!

**Comment by akitchen** [comment]
it sounds like what people are looking for would be covered by the current "Command Keys Only" option plus a new option to display navigation keys. This is probably simpler to add vs. an open-ended system for filtering which keys to display.
